index.js

```diff
@@ -38,7 +48,7 @@ module.exports = function getPlugin(S) {
      * Intercept the codeDeployLammbda hook and wrap the serverless handler
      * function in the configured wrapper function.
      */
-    onCodeDeployPre(evt) {
+    pre(evt) {
```

Comment:
Doesn't matter the name, but i personally like the full version of the func name than the shortened.

Reply:
Currently it applies to both _code deploy_ and _function run_, what name do you propose?

Reply:
I renamed them to `preAction` and `postAction`